daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md

```diff
@@ -28,146 +34,253 @@ Create a directory for your components file named `components` and inside it cre
+            var secret = await client.GetSecretAsync(SECRET_STORE_NAME, "secret");
+            Console.WriteLine($"Result: {string.Join(", ", secret)}");
+            secret = await client.GetSecretAsync(SECRET_STORE_NAME, "secret");
+            Console.WriteLine($"Result for bulk: {string.Join(", ", secret.Keys)}");
+        }
+    }
 }
-```
 
+```
 {{% /codetab %}}
 
 {{% codetab %}}
 
-```javascript
-require('isomorphic-fetch');
-const secretsUrl = `http://localhost:3500/v1.0/secrets`;
-
-fetch(`${secretsUrl}/my-secrets-store/my-secret`)
-        .then((response) => {
-            if (!response.ok) {
-                throw "Could not get secret";
-            }
-            return response.text();
-        }).then((secret) => {
-            console.log(secret);
-        });
-```
+```java
+
+//dependencies
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dapr.client.DaprClient;
+import io.dapr.client.DaprClientBuilder;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+
+//code
+@SpringBootApplication
+public class OrderProcessingServiceApplication {
+
+	private static final Logger log = LoggerFactory.getLogger(OrderProcessingServiceApplication.class);
+	private static final ObjectMapper JSON_SERIALIZER = new ObjectMapper();
+
+	private static final String SECRET_STORE_NAME = "localsecretstore";
+
+	public static void main(String[] args) throws InterruptedException, JsonProcessingException {
+		DaprClient client = new DaprClientBuilder().build();
+        //Using Dapr SDK to get a secret
+		Map<String, String> secret = client.getSecret(SECRET_STORE_NAME, "secret").block();
+		log.info("Result: " + JSON_SERIALIZER.writeValueAsString(secret));
+		try {
+			secret = client.getSecret(SECRET_STORE_NAME, "secret").block();
+			log.info("Result for random key: " + JSON_SERIALIZER.writeValueAsString(secret));
+		} catch (Exception ex) {
+			System.out.println("Got error for accessing key");
+		}
+	}
+}
 
+```
 {{% /codetab %}}
 
 {{% codetab %}}
 
 ```python
-import requests as req
 
-resp = req.get("http://localhost:3500/v1.0/secrets/my-secrets-store/my-secret")
-print(resp.text)
-```
+#dependencies 
+import logging
+from dapr.clients import DaprClient
+from dapr.clients.grpc._state import StateItem
+from dapr.clients.grpc._request import TransactionalStateOperation, TransactionOperationType
+
+#code
+logging.basicConfig(level = logging.INFO)
+    
+DAPR_STORE_NAME = "localsecretstore"
+key = 'secret'
+
+with DaprClient() as client:
+    #Using Dapr SDK to get a secret
+    secret = client.get_secret(store_name=DAPR_STORE_NAME, key=key)
+    logging.info('Result: ')
+    logging.info(secret.secret)
+    secret = client.get_bulk_secret(store_name=DAPR_STORE_NAME)
+    logging.info('Result for bulk secret: ')
+    logging.info(sorted(secret.secrets.items()))
+    try:
```

Comment:
can remove the try catch block .... its the same in this example as lines 200-203